Week1/mario-less/mario.c

```diff
@@ -0,0 +1,28 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    int n;
+    do
+    {
+        n = get_int("Size: ");
+    } while (n < 1 || n > 8);
+
+    for (int i = 0; i < n; i++)
+    {
+
+        for (int j = 0; j < n - i; j++)
+        {
+            printf(" ");
+        }
+
+        for (int j = 0; j < i + 1; j++)
+        {
+
+            printf("#");
+        }
+
+        printf("\n");
+    }
+}
```

Comment:
```suggestion
}

```

Rule of thumb: Always leave a blank line at the end of the file.